CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #2458, Fix a regression with the location header when inserting into views with PKs from multiple tables - @wolfgangwalther
  - #2356, Fix a regression in openapi output with mode follow-privileges - @wolfgangwalther
  - #2283, Fix infinite recursion when loading schema cache with self-referencing view - @wolfgangwalther
+ - #2481, Treat computed relationships not marked SETOF as M2O/O2M relationship instead of M2M - @wolfgangwalther
```

Comment:
Hm, shouldn't this be

```suggestion
 - #2481, Treat computed relationships not marked SETOF as M2O/O2O relationship instead of M2M - @wolfgangwalther
```

A `RETURNS table` always points to a to-one relationship right?

Reply:
Yeah, that's confusing. How about just removing the M2M mention(I've edited my suggestion above).

Reply:
And this should be a feature(in `### Added`) I think